test/utilities/workbench.ts

```diff
@@ -55,6 +55,13 @@ export async function zoomReset(wait: Duration = Duration.seconds(1)): Promise<v
   await executeQuickPick('View: Reset Zoom', wait);
 }
 
+export async function openNewTerminal(checkIfOpen = false): Promise<void> {
+  if (checkIfOpen) {
+    // TODO: do something here???
+  }
```

Comment:
I assume you have the if statement because you want to reuse the existing Terminal if one already exists?  In that case try `Output: Focus on Terminal View` for the TODO.  (By the way, `Output: Focus on Terminal View` does create a new Terminal if one does not already exist, so the if statement could be removed if this is indeed the intended use case.)

But if we do want to reuse Terminal windows, then the function name `openNewTerminal` sounds misleading, as it sounds like a new Terminal window should be created regardless if one already exists.

Reply:
Its a TODO for now.